### PR TITLE
Added button placeholders

### LINKED ISSUE #180: Implement gait event timeline

### PR DESCRIPTION
**Pull request by javitrejo14**
Closes #180 
Button and positions are set, no functionality yet.

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -403,8 +403,24 @@ body {
       #gait-suggestion-div {
          grid-row: 2; //timeline sub-grid
          grid-column: 2; //timeline sub-grid
-         //TODO <-----------------------------------------------------------------------------------
-
+         position: relative;
+         width: calc(100% - 12px);  //minus timeline track border and width of timeline thumb
```
Comment (PTLux1517) [inline src/App.scss:407]: heads up, the timeline track border is 3px, so this should be - 6px (two times it's thickness)

Comment (javitrejo14) [reply src/App.scss]: Fixed.

**Review by PTLux1517 — APPROVED** [review]

File: src/views/TimelineTrackView.tsx [hunk 2]
```diff
@@ -85,7 +85,9 @@ export default function TimelineTrackView(
             </div>
         </div>
         <div id={"gait-suggestion-div"}>
-            {/* TODO: add buttons */}
+            <button style={{left:"0%"}} />
+            <button style={{left:"calc(50% - 2px)"}} />
```
Comment (PTLux1517) [inline src/views/TimelineTrackView.tsx:89]: with your button width as 10px, you'll want to subtract half the button width to center it on the desired timeline percentage

Comment (PTLux1517) [reply src/views/TimelineTrackView.tsx:89]: this is fine for now though since these are only placeholders

Comment (javitrejo14) [reply src/views/TimelineTrackView.tsx:89]: Yes, that I was thinking just to get a visual of where they would be.

**Review by ConnorJacksonBSU — APPROVED** [review]

**Pull request merged**